src/components/QuickOpenModal.js

```diff
@@ -357,8 +357,23 @@ export class QuickOpenModal extends Component<Props, State> {
     return !this.getResultCount() && !!query;
   }
 
+  
+  renderSymLoading = (symbols: FormattedSymbolDeclarations, selectedSource: any, 
```

Comment:
Do you think we can extract these variables from props? Maybe we don't need to pass them here.